src/main/api/2/studio-api-2.yaml

```diff
@@ -4541,6 +4541,97 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /marketplace/remove:
+    post:
+      tags:
+        - marketplace
+      summary: Remove a plugin from the given site
+      description: 'Required permission "install_plugins"'
```

Comment:
Should we have a `remove_plugins` permission?

Reply:
I can think of the ability to install but not uninstall as a valid configuration. That gives some users the ability to add but not to remove/break this system. Please add it.